test/test_docstrings.py

```diff
@@ -26,6 +27,59 @@ def test_rst(s):
     assert not s or ret
 
 
+@given(text())
+def test_rst_extract_options_fuzz(s):
+    assume(not s.startswith(":"))
+    content, options = docstrings._rst_extract_options(s)
+    assert not options
+    assert content == s
+
+
+def test_rst_extract_options():
+    content = (
+        ":alpha: beta\n"
+        ":charlie:delta:foxtrot\n"
+        "rest of content\n"
+        ":option ignored: as follows content\n"
+    )
+    content, options = docstrings._rst_extract_options(content)
+    assert options == {
+        "alpha": "beta",
+        "charlie": "delta:foxtrot",
+    }
+    assert content == ("\nrest of content\n" ":option ignored: as follows content\n")
+
+
+@given(text())
+def test_rst_include_trim_fuzz(s):
```

Comment:
See above - given that `_rst_include_trim` may also raise for some inputs, we can't really do anything meaningful. I propose we skip this test.